cEP-0020.md

```diff
@@ -0,0 +1,418 @@
+
+Some barriers they face include orientation issues that can potentially
+demotivate newcomers from placing their first contribution. On the other
+hand, gamification is widely used to engage and motivate people to
+accomplish tasks and improve their performance.
+
+Therefore, the goal of this project is to use gamification to orient and
+motivate newcomers to overcome onboarding barriers to contribute and engage
+with coala and its community.
+
+## Gamification Introduction
+
+There will be four game design elements to help newcomers: Activities, Points,
+Badges and Levels. For each of these elements, we will have a set of rules to
+describe the operation of the gamified environment. i.e.:
+
+### Activities
+
+Let's start by defining some activities which could be tracked:
+
+- Create a bug report
+- Create newcomer difficulty issue
+- Create low difficulty issue
+- Create medium difficulty issue
+- Create a bear proposal issue
+- Create pull request that fixes a bug
+- Create pull request for bear proposal
+- Create pull request for newcomer issue
+- Create pull request for low difficulty issue
+- Create pull request for medium difficulty issue
+- Gitmate errors on pull request
+- CI status of the pull request
+- Review comments on pull request
+- Messages on Gitter channels
+  - Good messages
+  - Bad messages
+- Introduce `.coafile` in other projects
+
+### Points
+
+Points are a simple tool to reward behavior, and they provide an excellent
+base upon which to build a gamified design. Now let's assign some points
+to each of the tracked activities mentioned above:
+
+- Create Issue
+  - Points will only be given when the issue has an extra label `process/verified`
+    and that label can only be added by a maintainer.
+    - 20 points for type/bug
+    - 20 points for bear proposal
+    - 10 points for newcomer difficulty and then increase of 10 points with the
+      increase of difficulty levels
+- Create Pull Requests
+  - No points given until all the CI on the pull request is passing
+  - If CI passes and there are no Gitmate errors
+    - 20 points for bug fixes
+    - 20 points for writing a bear
+    - 10 points for newcomer difficulty and then increase of 10 points with the
+      increase in difficulty levels
+  - If CI passes and Gitmate errors are b/w 1 to 5
+    - 10 points for bug fixes
+    - 10 points for writing a bear
+    - 5 points for intial and then increase
+  - If CI passes and Gitmate errors are b/w 5 to 10
+    - 5 points for bug fixes
+    - 5 points for writing a bear
+    - 3 points for intial and then increase
+  - If CI passes and Gitmate errors are b/w 10 and beyond
+    - No points given
+  - If the pull request gets merged
+    - 5 points for merging any pull request
+  - If the pull request is closed without merged
+    - All the points given for that pull request will be reset
+- Review comments on the pull request
+  - 2 points per good review comments
```

Comment:
you will not be responsible for defining the scoring of review comments.
That is clearly the responsibility of the meta-review project.
This should be clearly explained in your cEP that it is not in-scope for your project.

Reply:
The meta-review project will provide necessary data (reactions to the review comment) in community repo, but that project itself doesn't define `scoring of review comments`. I think this project should use that data (reactions) to calculate scoring.